GooglePlayInstant/Editor/AppBundlePublisher.cs

```diff
@@ -15,6 +15,13 @@
 using UnityEditor;
 using UnityEngine;
 
+// In the Unity 2017 series the EditorUserBuildSettings.buildAppBundle field was introduced in 2017.4.17.
+// It might seem preferable to modify buildAppBundle using reflection, but the field is extern.
+// Instead check for quite a few versions in the 2017.4.17+ series.
+#if UNITY_2018_3_OR_NEWER || UNITY_2017_4_17 || UNITY_2017_4_18 || UNITY_2017_4_19 || UNITY_2017_4_20 || UNITY_2017_4_21 || UNITY_2017_4_22 || UNITY_2017_4_23 || UNITY_2017_4_24 || UNITY_2017_4_25 || UNITY_2017_4_26 || UNITY_2017_4_27 || UNITY_2017_4_28 || UNITY_2017_4_29
```

Comment:
I think it's a fair assumption that Unity is never going to have more than 29 subversions, but it might be good to mention that assumption explicitly in the comments.

Reply:
Good suggestion, done.

I also went ahead and added up through UNITY_2017_4_40 just to make it more likely that we're covered.